doc/source/whatsnew/v0.21.0.txt

```diff
@@ -147,7 +147,8 @@ Bug Fixes
 Conversion
 ^^^^^^^^^^
 
-
+- :func:`dtypes.common._get_dtype` now accepts ``Categorical``s and
```

Comment:
not this doesn't need a note at all. This is not a user relevant change as ``_get_dtype`` is not user facing.